src/modules/ekf2/EKF/python/ekf_derivation/derivation.py

```diff
@@ -101,7 +99,7 @@ def predict_covariance(
     p = sf.V3(state[State.px], state[State.py], state[State.pz])
 
     q_new = q * sf.Quaternion(sf.V3(0.5 * d_ang_true[0],  0.5 * d_ang_true[1],  0.5 * d_ang_true[2]), 1)
-    v_new = v + R_to_earth * d_vel_true + sf.V3(0 ,0 ,g) * dt
```

Comment:
Do you think it would be better to remove the position states completely from the estimates? or do we want this configurable?

Reply:
I just removed the gravity component. But if we make it configurable then it can be user selected with a parameter on running the script.